lib/reports/ppl-sla/index.js

```diff
@@ -19,11 +20,11 @@ module.exports = ({ db, query: params, flow }) => {
   const parse = record => {
 
     const getDeadlineState = () => {
-      const currentState = record.activity.reduce((state, activity) => {
+      const currentState = sortBy(record.activity, 'created_at', 'asc').reduce((state, activity) => {
         const status = flow[activity.event.status];
         state.extended = state.extended || get(activity, 'event.data.extended', false);
         const isSubmission = !state.withASRU && status.withASRU;
-        const isClosedOrReturned = state.withASRU && !status.withASRU;
+        const isClosedOrReturned = !status.open || (state.withASRU && !status.withASRU);
```

Comment:
This is a no-op as as all closed statuses are `withASRU: false` but it makes it a bit clearer.